qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteDiagnostics.java

```diff
@@ -703,6 +703,8 @@ private ResolvedJavaTypeInfo validatePropertyPart(PropertyPart part, Section own
 			Range range = QutePositionUtility.createRange(part);
 			Diagnostic diagnostic = createDiagnostic(range, DiagnosticSeverity.Error, QuteErrorCode.UnknownProperty,
 					part.getPartName(), baseType.getSignature());
+			// Create data information helpful for code action
+			diagnostic.setData(DiagnosticDataFactory.createUnknownPropertyData(part.getPartName()));

```

Comment:
I suggest that you see @datho7561 pr which stores the signature. I think it will help you to know the base type.